intera_tools_description/urdf/electric_gripper/rethink_electric_gripper.xacro

```diff
@@ -108,7 +108,7 @@
       <parent link="${side}_electric_gripper_base"/>
       <child link="${gripper_side}_gripper_l_finger"/>
       <limit effort="20.0" lower="0.0" upper="0.020833" velocity="5.0"/>
-      <dynamics damping="0.7" friction="0.0"/>
+      <dynamics damping="1000.0" friction="5000.0"/>
```

Comment:
Do these numbers seem ... large? I don't really have a good frame of reference, but they just seem like large numbers to me, and thus maybe deserve a comment (at the very least, in this PR or the commit description).

Reply:
Yeah, this is rough. The comment is that the worm gear is not back-drivable. Without the damping and friction params, the fingers will wobble when moving the arm around.